BREAKING.md

```diff
@@ -10,6 +10,23 @@ There are a few steps you can take to write a good change note and avoid needing
 - Provide guidance on how the change should be consumed if applicable, such as by specifying replacement APIs.
 - Consider providing code examples as part of guidance for non-trivial changes.
 
+## 0.53 Breaking changes
```

Comment:
Given that you are not changing any public interface, I do not see any reason to put any comment here at all. people should not use RemoteFluidObjectHandle directly, and it would be great to confirm it is not exposed from package (I believe it's the case given that this change id not generate change in .api files)

Reply:
`path()` is not exposed from `@fluidframework/runtime-utils` package